client.go

```diff
@@ -592,8 +592,11 @@ func (f *File) Stat() (os.FileInfo, error) {
 	return fileInfoFromStat(fs, path.Base(f.path)), nil
 }
 
+// clamp reads to less than 32k
+var MaxReadPacket = 32 * 1024
+
 // clamp writes to less than 32k
-const maxWritePacket = 1 << 15
+var MaxWritePacket = 32 * 1024
```

Comment:
Now that this variable is exported, it should be better documented.

The comment should begin with `// MaxWritePacket ...` (see [here](https://github.com/golang/go/wiki/CodeReviewComments#doc-comments)) and explain what it does, e.g., `// MaxWritePacket controls the maximum size in bytes of a single write packet.` or so.